ext/odbc/odbc.stub.php

```diff
@@ -4,193 +4,193 @@
-function odbc_result($result_id, string|int $field): string|bool|null {}
+/** @param resource $stmt */
+function odbc_result($stmt, string|int $column): string|bool|null {}
 
-/** @param resource $result_id */
-function odbc_result_all($result_id, string $format = ""): int|false {}
+/** @param resource $stmt */
+function odbc_result_all($stmt, string $format = ""): int|false {}
 
-/** @param resource $result_id */
-function odbc_free_result($result_id): bool {}
+/** @param resource $stmt */
+function odbc_free_result($stmt): bool {}
 
 /** @return resource|false */
 function odbc_connect(string $dsn, string $user, string $password, int $cursor_option = SQL_CUR_USE_DRIVER) {}
 
 /** @return resource|false */
 function odbc_pconnect(string $dsn, string $user, string $password, int $cursor_option = SQL_CUR_USE_DRIVER) {}
 
-/** @param resource $connection_id */
-function odbc_close($connection_id): void {}
+/** @param resource $odbc */
+function odbc_close($odbc): void {}
 
-/** @param resource $result_id */
-function odbc_num_rows($result_id): int {}
+/** @param resource $stmt */
+function odbc_num_rows($stmt): int {}
 
 #if !defined(HAVE_SOLID) && !defined(HAVE_SOLID_30)
-/** @param resource $result_id */
-function odbc_next_result($result_id): bool {}
+/** @param resource $stmt */
+function odbc_next_result($stmt): bool {}
 #endif
 
-/** @param resource $result_id */
-function odbc_num_fields($result_id): int {}
+/** @param resource $stmt */
+function odbc_num_fields($stmt): int {}
 
-/** @param resource $result_id */
-function odbc_field_name($result_id, int $field_number): string|false {}
+/** @param resource $stmt */
+function odbc_field_name($stmt, int $field_number): string|false {}
 
-/** @param resource $result_id */
-function odbc_field_type($result_id, int $field_number): string|false {}
+/** @param resource $stmt */
+function odbc_field_type($stmt, int $field_number): string|false {}
 
-/** @param resource $result_id */
-function odbc_field_len($result_id, int $field_number): int|false {}
+/** @param resource $stmt */
+function odbc_field_len($stmt, int $field_number): int|false {}
 
 /**
- * @param resource $result_id
+ * @param resource $stmt
  * @alias odbc_field_len
  */
-function odbc_field_precision($result_id, int $field_number): int|false {}
+function odbc_field_precision($stmt, int $field_number): int|false {}
 
-/** @param resource $result_id */
-function odbc_field_scale($result_id, int $field_number): int|false {}
+/** @param resource $stmt */
+function odbc_field_scale($stmt, int $field_number): int|false {}
 
-/** @param resource $result_id */
-function odbc_field_num($result_id, string $field_name): int|false {}
+/** @param resource $stmt */
+function odbc_field_num($stmt, string $field_name): int|false {}
 
-/** @param resource $connection_id */
-function odbc_autocommit($connection_id, int $onoff = 0): int|bool {}
+/** @param resource $odbc */
+function odbc_autocommit($odbc, int $enable = 0): int|bool {}
```

Comment:
We should change this to a bool argument...

Reply:
If it's not too late for 8.0, I'll gladly do this. We should change the `$reverse` parameter of `imap_sort()` to bool as well.

Reply:
Well, I did a few int -> bool changes recently, so... :)

I'd suggest to land those separately though, to make sure they're in rc2.

Reply:
Done as PR #6328.